tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2015",
+    "target": "es5",
```

Comment:
Is there a reason for changing this? We would want to use features from 2015

Reply:
Hmmm, yes this was related to the leaflet issues. I'm going to check this once more in case it's not needed.

Reply:
reverted